packages/edge_runtime/lib/src/response.dart

```diff
@@ -9,57 +9,102 @@ import 'blob.dart';
 import 'body.dart';
 import 'form_data.dart';
 import 'headers.dart';
+import 'interop/headers.dart' as headers_interop;
 import 'interop/readable_stream.dart';
 import 'interop/utils_interop.dart';
-import 'interop/headers.dart' as headers_interop;
 
 class Response implements Body {
   final interop.Response _delegate;
 
   Response._(this._delegate);
 
+  /// Creates a new [Response] object.
+  /// The [body] can be a [String], [ByteBuffer] or [Uint8List].
   Response(
     Object? body, {
-    int? status,
-    String? statusText,
+    int status = 200,
```

Comment:
Nit but this is technically a breaking change

Reply:
Folks could be doing:

```dart
class MyResponse {
  int? status;
  
  Response toResponse() {
    return Response(..., status: status);
  }
}
```

Such change would break it.

Reply:
Correct, will revert it :)